src/screens/Topics/index.js

```diff
@@ -187,39 +190,40 @@ const Topics = () => {
   );
 };
 const styles = StyleSheet.create({
-  scrollViewStyle: {marginBottom: 100},
+  scrollViewStyle: (bottom) => ({
+    marginBottom: dimen.normalizeDimen(100) - bottom
+  }),
   header: {
     flexDirection: 'row',
     justifyContent: 'space-between',
     alignItems: 'center'
   },
   container: {
     flex: 1,
-    padding: Platform.OS === 'ios' ? 22 : 0,
+    padding: Platform.OS === 'ios' ? dimen.normalizeDimen(22) : 0,
     backgroundColor: colors.white
   },
   containerProgress: {
-    marginTop: 20,
-    marginBottom: 24,
-    paddingHorizontal: 20
+    marginTop: dimen.normalizeDimen(20),
+    marginBottom: dimen.normalizeDimen(24),
+    paddingHorizontal: dimen.normalizeDimen(20)
   },
   textPickYourTopic: {
     fontFamily: 'Inter-Bold',
     fontStyle: 'normal',
     fontWeight: '600',
-    fontSize: 36,
-    lineHeight: 43.57,
+    fontSize: normalizeFontSize(36),
+    lineHeight: normalizeFontSize(43.57),
     color: '#11243D',
-    marginHorizontal: 20
+    marginHorizontal: dimen.normalizeDimen(20)
   },
   footer: {
     position: 'absolute',
     bottom: 0,
-    height: 112,
+    height: dimen.normalizeDimen(112),
     width,
-    paddingLeft: 20,
-    paddingRight: 20,
-    paddingBottom: 20,
+    paddingHorizontal: dimen.normalizeDimen(20),
+    paddingBottom: dimen.normalizeDimen(20),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for improving the responsiveness of the application. The use of `dimen.normalizeDimen` function to normalize dimensions is a good practice in React Native to ensure consistent layout across different screen sizes and resolutions.



<!-- This is an auto-generated comment by OSS CodeRabbit -->